test/README.md

```diff
@@ -99,29 +98,59 @@ The JSON tests `db/json` are executed on 3 standard files (1 ELF, 1 MachO, 1 PE)
 # Commands tests
 
 Example commands tests for the other `db/` folders:
+```sh
+NAME=test_db
+FILE=bins/elf/ls
+CMDS=<<EOF
+pd 4
+EOF
+EXPECT=<<EOF
+        ;-- main:
+        ;-- entry0:
+        ;-- func.100001174:
+        0x100001174      55             Push rbp
+        0x100001175      4889e5         Mov  rbp, rsp
+        0x100001178      4157           Push r15
+EOF
+RUN
+```
+It is also possible to match specific parts of the output in `EXPECT` and `EXPECT_ERR` using
+regex (with `REGEXP_OUT` and `REGEXP_ERR` respectively) in case some of the test's output is dynamic:
+```sh
+NAME=bp rebase
+FILE=bins/elf/analysis/pie
+ARGS=-d
+CMDS=<<EOF
+aa
+db main
+db~main
+doc
+db~main
+EOF
+REGEXP_OUT=([a-zA-Z="]+\s+)
```

Comment:
I'm not sure I understand how this work.. Does this regex removes every char from output that is not a-z, A-Z, =, ", spaces ?

Reply:
it only matches a sequence a-z, A-Z, =, " followed by whitespace
Since we don't have negative lookahead these things get a bit tricky.. Will a simpler example be better?

Reply:
Sorry, maybe it's just me who does not understand....

How this regex "converts"
```
 0x566495c5 - 0x566495c6 1 --x sw break enabled valid cmd="" cond="" name="main" module="/home/user/rizin/test/bins/elf/analysis/pie"
```
into 
```
x sw break enabled valid cmd="" cond="" name="main" pie"
```
?

What is more or less the operation that rz-test does if you would write it in python with the `re` module? (if you know python, otherwise just explaining in words could be fine). Sorry again, but I want to use this regex thing but i'm still not sure I understand how.

Reply:
The regex is applied and all matches are printed as a single output. I tried playing with having the user specify the number of matches but it felt error prone. This format seems intuitive to me since that's what I expect when I regex in a text editor/grep, we can improve the documentation or test format if it isn't clear.

```python
>>> import re
>>> string = "0x566495c5 - 0x566495c6 1 --x sw break enabled valid cmd=\"\" cond=\"\" name=\"main\" module=\"/home/user/rizin/test/bins/elf/analysis/pie\""
>>> res = re.findall("([a-zA-Z=\"]+\s+)", string)
>>> print(res)
['x ', 'sw ', 'break ', 'enabled ', 'valid ', 'cmd="" ', 'cond="" ', 'name="main" ']
>>> expect = "".join(res)
'x sw break enabled valid cmd="" cond="" name="main" '
>>> print(expect)
```

You will also get the same result with
```sh
echo "0x566495c5 - 0x566495c6 1 --x sw break enabled valid cmd=\"\" cond=\"\" name=\"main\" module=\"/home/user/rizin/test/bins/elf/analysis/pie\"" | grep -E "([a-zA-Z=\"]+\s+)"
```

the string `pie"`(the only part of that word which started with letters and ends with a whitespace) isn't matched in these example because the given strings were terminated without a whitespace.